src/tests/build.proj

```diff
@@ -427,7 +427,10 @@
       <_LinkerFlagsToDrop Include="@(NativeFramework->'-framework %(Identity)')" />
       <LinkerArg Remove="@(_LinkerFlagsToDrop)" />
       <ExtraAppLinkerArgs Include="@(LinkerArg)" />
-      <GlobalizationNativeLibs Include="$(MicrosoftNetCoreAppRuntimePackDir)runtimes/$(TargetOS)-$(TargetArchitecture)/native/icudt.dat" />
+      <GlobalizationNativeLibs Include="$(MicrosoftNetCoreAppRuntimePackDir)runtimes/$(TargetOS)-$(TargetArchitecture)/native/icudt.dat;
```

Comment:
Looks good! Can you push your changes to the mirror azdo dotnet/runtime and invoke the iOS perf jobs in the https://dev.azure.com/dnceng/internal/_build/results?buildId=2233228&view=results? 

I don't think any additional changes are needed for the sample app, but would be good to confirm that.

Reply:
https://dev.azure.com/dnceng/internal/_build/results?buildId=2233344&view=results
When it will finish, we can check and proceed with merging this PR.